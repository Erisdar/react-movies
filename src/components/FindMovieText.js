import React from 'react';

export default () => (
    <p className="find-movie-text">FIND YOUR MOVIE</p>
);
