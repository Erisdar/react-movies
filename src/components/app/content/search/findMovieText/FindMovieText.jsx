import React from 'react';
import './findMovieText.css';

export default () => (
    <p className="find-movie-text">FIND YOUR MOVIE</p>
);
