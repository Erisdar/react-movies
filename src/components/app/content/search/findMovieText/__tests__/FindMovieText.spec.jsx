import { shallow } from 'enzyme';
import FindMovieText from '../FindMovieText';
import React from 'react';

describe('FindMovieText', () => {
    it('should be render correctly', () => {
        const component = shallow(<FindMovieText />);
        expect(component).toMatchSnapshot();
    });
});