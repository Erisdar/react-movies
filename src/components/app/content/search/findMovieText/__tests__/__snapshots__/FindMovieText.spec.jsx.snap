// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FindMovieText should be render correctly 1`] = `
<p
  className="find-movie-text"
>
  FIND YOUR MOVIE
</p>
`;
